### PR TITLE
Add prompt-stash extension

### PR DESCRIPTION
**Pull request by renzo4web**
## Description

# Merge Request Description

A Raycast extension that provides CRUD operations for managing AI prompts. It allows users to store, edit, and retrieve prompts with tags for any LLM platform, using local storage for data persistence.

## Technical Scope
- Prompt storage and retrieval
- Tag-based organization
- Search functionality
- Clipboard integration

## Testing Notes
The extension has been tested for:
- Prompt creation and editing
- Tag management
- Search functionality
- Keyboard shortcuts
- Clipboard operations
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for your contribution. It looks like it's doing something similar to what is [done here](https://www.raycast.com/HelloImSteven/promptlab) by @SKaplanOfficial

**Comment by renzo4web** [comment]
> Thanks for your contribution. It looks like it's doing something similar to what is [done here](https://www.raycast.com/HelloImSteven/promptlab) by @SKaplanOfficial

Thank you for your feedback and for mentioning  **PromptLab** extension.

While both **PromptStash** and **PromptLab** manage AI prompts, **PromptStash** is designed to be much simpler and more user-friendly. It allows users to quickly save, organize, and access their prompts without the complexity of advanced configurations or additional features. 

**Key Differences:**

- **Ease of Use**: 
  - **PromptStash** offers a straightforward interface for rapid prompt management.
  - **PromptLab** includes extensive customization options which can be overwhelming for users seeking simplicity.

- **Configuration**:
  - **PromptStash** requires minimal setup, making it ideal for quick prompt saving.
  - **PromptLab** involves complex configurations with placeholders and action scripts, which might be unnecessary for users who only need basic prompt storage.

- **Target Audience**:
  - **PromptStash** is perfect for users who need an efficient way to manage prompts without additional setup.
  - **PromptLab** caters to power users who need advanced features and are comfortable with more intricate configurations.

We believe both extensions serve different user needs.

Thanks again for your support!

File: extensions/prompt-stash/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              />
+              <Action.Push
+                icon={Icon.Eye}
+                onPop={loadPrompts}
+                title="View Prompt"
+                target={<PromptDetail prompt={prompt} />}
+              />
+              <Action
+                icon={Icon.Trash}
+                title="Delete Prompt"
```
Comment (pernielsentikaer) [inline extensions/prompt-stash/src/index.tsx:71]: Use `shortcut={Keyboard.Shortcut.Common.Remove}` and `style={Action.Style.Destructive}`

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/prompt-stash/src/index.tsx (hunk 1 above)
Comment (renzo4web) [reply extensions/prompt-stash/src/index.tsx:71]: @pernielsentikaer  Changes have been made, thank you for your feedback!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by renzo4web** [comment]
@pernielsentikaer thank you, When is this going to be merged?

**Pull request merged**